lib/tasks/assets.rake

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+namespace :assets do
+  desc 'Copy raw assets'
+  task :copy_raw_assets do
+    assets_path = File.join(Rails.root, 'app', 'assets', 'raw')
+    assets = Dir.glob("#{assets_path}/**/**", File::FNM_DOTMATCH).select.map do |file|
```

Comment:
Do we need the `.select` method call in this line?

Reply:
Nope I will remove it 👍